site/src/components/DeploySettingsLayout/OptionsTable.tsx

```diff
@@ -46,6 +49,29 @@ const OptionsTable: FC<{
                 <TableCell>
                   <OptionName>{option.name}</OptionName>
                   <OptionDescription>{option.description}</OptionDescription>
+                  <Box
+                    sx={{
+                      marginTop: 3,
+                      display: "flex",
+                      flexWrap: "wrap",
+                      gap: 1,
+                    }}
+                  >
+                    <OptionConfig>
+                      <OptionConfigFlag>CLI</OptionConfigFlag>
+                      --{option.flag}
+                    </OptionConfig>
+                    {option.flag_shorthand && (
+                      <OptionConfig>
+                        <OptionConfigFlag>CLI</OptionConfigFlag>
+                        --{option.flag_shorthand}
```

Comment:
I think shorthand flags always use a single dash.

```suggestion
                        -{option.flag_shorthand}
```